services/brig/src/Brig/API.hs

```diff
@@ -1076,7 +1076,7 @@ getRichInfo (self ::: user ::: _) = do
         (Just t1, Just t2) | t1 == t2 -> pure ()
         _ -> throwStd insufficientTeamPermissions
     -- Query rich info
-    json . fromMaybe emptyRichInfo <$> lift (API.lookupRichInfo user)
+    json . fromMaybe (RichInfoAssocList []) <$> lift (API.lookupRichInfo user)
```

Comment:
we really should split these functions up into a typed core and a wrapper that only does the json conversion.  yeay servant!  :)